canbench-bin/src/table.rs

```diff
@@ -88,18 +88,14 @@ pub(crate) fn print_table(data: &[Entry]) {
     }
 
     // Calculate max column widths
-    let mut col_widths = columns
-        .iter()
-        .map(|header| header.len())
-        .collect::<Vec<_>>();
-
+    let mut col_widths = columns.iter().map(|h| h.len()).collect::<Vec<_>>();
```

Comment:
This misses the fact that not all rows are actually printed.

Reply:
good point, thank you! fixed.